Source/FullScreenPokemon.ts

```diff
@@ -4942,11 +4962,14 @@ module FullScreenPokemon {
          * @param move   The move being used.
          * @todo Eventually add check to make sure the Player beat the Gym leader needed to use the move.
          * @todo Add context for what happens if player is not bordering the correct HMCharacter.
+         * @todo Refactor to give borderedThing a .hmActivate property.
          */
         partyActivateCheckThing(player: IPlayer, pokemon: IPokemon, move: IMoveSchema): void {
             var borderedThing: IThing = player.bordering[player.direction];
-
-            if (borderedThing && borderedThing.title === move.characterName) {
+            if (move.characterName === undefined) {
```

Comment:
Fly doesn't have an object to interact with. In the future this will probably need to change to take into account flash